### PR TITLE
Add application management simulation

### PR DESCRIPTION
**Pull request by shakeelmohamed**

### CODE REVIEW

**Comment by shakeelmohamed** [comment]
@tdhellmann That makes sense, I've added some example searches to the README

File: 02 app management/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Application Management example
+
+This simulation has CSV files used for mockdata. They are included under the `data` directory here. CSV mockdata files must reside at the same file system path as the simulation file, or the `data` directory at that path.
```
Comment (PKing70) [inline 02 app management/README.md:3]: `mockdata`
```suggestion
This simulation has CSV files used for `mockdata`. They are included under the `data` directory here. CSV `mockdata` files must reside at the same file system path as the simulation file, or the `data` directory at that path.
```

File: 02 app management/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+The application management simulation intends to model a simple web application with 3 entities:
```
Comment (PKing70) [inline 02 app management/README.md:5]:
```suggestion
The application management simulation intends to model a simple web application with three entities:
```

File: 02 app management/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+* `User` entities sending requests to `WebServer` entities
+* `WebServer` entities receiving requests from `User` entities, and sending/receiving requests to `Database` entities
+* `Database` entities receiving requests from `WebServer` entities
+
+## Sending data to Splunk Enterprise HEC
+
+Before using this simulation some configuration needs to be updated.
```
Comment (PKing70) [inline 02 app management/README.md:13]:
```suggestion
Before using this simulation, some configuration files need to be updated.
```

File: 02 app management/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+In `app-man.json`, under the `transports` property there are some placeholder values for the `SplunkHEC` transport.
```
Comment (PKing70) [inline 02 app management/README.md:15]:
```suggestion
In `app-man.json`, under the `transports` property, there are some placeholder values for the `SplunkHEC` transport.
```

File: 02 app management/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+* `uri` is the Splunk Enterprise URI including the HEC port (8088 by default)
+* `token` is the Splunk Enterprise HEC token
+* `default_index` is the default Splunk Enterprise index to send events to
+
+## Setup
+
+This simulation will generate a good amount of data in the index configured in the `app-man.json` file. Just make sure the index exists, and the Splunk HEC token is configured to send data to that index.
```
Comment (PKing70) [inline 02 app management/README.md:23]:
```suggestion
This simulation will generate data in the index configured in the `app-man.json` file. Make sure the index exists, and the Splunk HEC token is configured to send data to that index.
```

File: 02 app management/README.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+Since events are configured to go to `SplunkHEC`, nothing will be printed to the console.
+
+```sh
+Starting simulation
+```
+
+### Helpful Splunk searches
+
+The following queries are some examples for finding meaning in the generated data through Splunk searches.
```
Comment (PKing70) [inline 02 app management/README.md:41]:
```suggestion
The following queries show examples of finding meaningful information in the generated data using Splunk searches.
```

File: 02 app management/README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
+# Application Management example
```
Comment (PKing70) [inline 02 app management/README.md:1]:
```suggestion
# Application management example
```

File: 02 app management/README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+```sh
+Starting simulation
+```
+
+### Helpful Splunk searches
+
+The following queries are some examples for finding meaning in the generated data through Splunk searches.
+
+#### HTTP Status Error codes by Webserver
```
Comment (PKing70) [inline 02 app management/README.md:43]:
```suggestion
#### HTTP status error codes by webserver
```

File: 02 app management/app-man.simulation [hunk 9]
```diff
@@ -0,0 +1,379 @@
...
+                connectionID: message.connectionID;
+                status: databaseStatus;
+                userRef: message.userRef;
+                queryTime: 0;
+                renderTime: message.renderTime;
+                pageName: message.pageName;
+                webserverName: message.webserverRef;
+            }
+
+            // Reduce the number of database server down message
```
Comment (PKing70) [inline 02 app management/app-man.simulation:365]:
```suggestion
            // Reduce the number of database server down messages
```

File: 02 app management/README.md (hunk 7 above)
Comment (shakeelmohamed) [reply 02 app management/README.md:1]: leaving the M capitalized as it's the name of the example

**Pull request merged**